src/main/java/ch/njol/skript/test/runner/ExprRuntimeErrors.java

```diff
@@ -0,0 +1,49 @@
+package ch.njol.skript.test.runner;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.NoDoc;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@NoDoc
+public class ExprRuntimeErrors extends SimpleExpression<String> {
+
+	static {
+		if (Skript.testing())
```

Comment:
This isn't what you want. testing() includes debug verbosity. You want TestMode.ENABLED.